index.html

```diff
@@ -574,7 +575,8 @@ <h2 class="bannerbox__title">2016 Election Countdown Clock</h2>
         <li class="available-times__row">
             <div class="form-group">
                 <label for="{{type}}Date{{id}}">Date</label>
-                <input class="form-input input--date" type="date" name="{{type}}Date" id="{{type}}Date{{id}}" min="2016-10-20" max="2016-11-08" required />
+                <input type="text" class="form-input input--date" type="date" id="{{type}}Date{{id}}" required> 
+                <input type="hidden" class="input-formatted-date" name="{{type}}Date" id="alt-{{type}}Date{{id}}">
```

Comment:
I'd much much prefer to keep these as type="date" and only use "text" where date isn't supported. The date input is much more accessible and works much more nicely across different platforms - especially on mobile.